com.github.thezbyg.gpick.yaml

```diff
@@ -11,6 +11,8 @@ finish-args:
   # X11 + XShm access
   - --share=ipc
   - --socket=x11
+  # Filesystem access
+  - --filesystem=host
```

Comment:
Why it does requires host filesystem access?

Reply:
@bilelmoussaoui 

> Why it does requires host filesystem access?

To save to and read from files (Gpick palette, GIMP/Inkscape palette and Adobe Swatch Exchange format).

Please keep in mind that the `host` option does not actually provide complete access to the host filesystem. It offers very similar functionality to `home`, but with the addition of access to `/run/media`.

Reply:
I'd prefer to see it limited to xdg-pictures. It's easy for users to change it themselves.

Reply:
@barthalion 

This app is intended to read files created by GIMP and Inkscape, and they both have the `host` filesystem privileges.
https://github.com/flathub/org.inkscape.Inkscape/blob/4d9be1cd3d918bad54463b800494ec2c415ec2b5/org.inkscape.Inkscape.json#L13
https://github.com/flathub/org.gimp.GIMP/blob/b43dc36fbcbc1034b54f701d96616d5954e60277/org.gimp.GIMP.json#L13

Even the EOG (Eye of GNOME) has the same permissions, and Ciano has access at least to the whole `home` directory.
https://github.com/flathub/org.gnome.eog/blob/496799ee3d8e788fe60fad452206e8eb12445633/org.gnome.eog.yml#L18
https://github.com/flathub/com.github.robertsanseries.ciano/blob/c2a3f7a0b63b1868f3c39325b4021ef76f162971/com.github.robertsanseries.ciano.json#L19

By limiting access to `~/Pictures` directory, we make this application useless for most people.